src/chrome/content/rules/Threema.xml

```diff
@@ -1,27 +1,26 @@
 <ruleset name="Threema">
-    <target host="threema.ch" />
-    <target host="www.threema.ch" />
-    <target host="gateway.threema.ch" />
-    <target host="shop.threema.ch" />
-    <target host="myid.threema.ch" />
-    <target host="avatar.threema.ch" />
-    <target host="work.threema.ch" />
-    <target host="three.ma" />
-    <target host="web.threema.ch" />
-    <target host="web-beta.threema.ch" />
+	<target host="threema.ch" />
+	<target host="www.threema.ch" />
+	<target host="gateway.threema.ch" />
+	<target host="shop.threema.ch" />
+	<target host="myid.threema.ch" />
+	<target host="avatar.threema.ch" />
+	<target host="work.threema.ch" />
+	<target host="three.ma" />
+	<target host="web.threema.ch" />
+	<target host="web-beta.threema.ch" />
 
-    <test url="http://threema.ch/en/whats-new" />
-    <test url="http://gateway.threema.ch/en/developer/api" />
-    <test url="http://shop.threema.ch/download" />
-    <test url="http://myid.threema.ch/revoke" />
-    <test url="http://myid.threema.ch/unlink" />
-    <test url="http://avatar.threema.ch/*THREEMA" />
-    
-    <!-- The main page serves a 403 reply -->
-    <exclusion pattern="http://avatar\.threema\.ch/$" />
+	<test url="http://threema.ch/en/whats-new" />
+	<test url="http://gateway.threema.ch/en/developer/api" />
+	<test url="http://shop.threema.ch/download" />
+	<test url="http://myid.threema.ch/revoke" />
+	<test url="http://myid.threema.ch/unlink" />
+	<test url="http://avatar.threema.ch/*THREEMA" />
 
-    <securecookie host="^(www.|gateway.|work.|shop\.|myid.|web.|web-beta.)?threema\.ch" name=".+" />
+	<!-- The main page serves a 403 reply -->
+	<exclusion pattern="http://avatar\.threema\.ch/$" />
```

Comment:
I think this exclusion can be removed because both HTTP and HTTPS return the same code.